#前端配置文件
data_path: ./

registration_method: NDT

#local map
key_frame_distance: 2.0
local_frame_num: 20
local_map_filter: voxel_filter

#rviz
#display_filter: voxel_filter

#current frame
frame_filter: voxel_filter

#registration config
NDT:
  res: 1.0
  step_size: 0.1
  trans_eps: 0.01
  max_iteration: 30

#Filter
#冒号和数据之间要有空格
voxel_filter:
  local_map:
    leaf_size: [0.6,0.6,0.6]
  frame:
    leaf_size: [1.3,1.3,1.3]
  # display:
  #   leaf_size: [0.5,0.5,0.5]
